### PR TITLE
add: entry permission

### LINKED ISSUE #313: add: entry permissions

### PR DESCRIPTION
**Pull request by ChienNQuang**

### CODE REVIEW

File: src/Application/Common/Models/Operations/EntryOperation.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace Application.Common.Models.Operations;
+
+public enum EntryOperation
+{
+    View,
+    Upload,
+    Download,
+    ChangePermission
```
Comment (kaitoz11) [inline src/Application/Common/Models/Operations/EntryOperation.cs:8]: Can we add `Share` here?

File: src/Infrastructure/Persistence/Configurations/EntryPermissionConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Infrastructure.Persistence.Configurations;
+
+public class EntryPermissionConfiguration : IEntityTypeConfiguration<EntryPermission>
+{
+    public void Configure(EntityTypeBuilder<EntryPermission> builder)
+    {
+        builder.HasKey(x => new { x.EntryId, x.EmployeeId });
```
Comment (kaitoz11) [inline src/Infrastructure/Persistence/Configurations/EntryPermissionConfiguration.cs:11]: I didn't know we can do this.

**Review by kaitoz11 — APPROVED** [review]
LGTM

File: src/Infrastructure/Persistence/Configurations/EntryPermissionConfiguration.cs (hunk 2 above)
Comment (ChienNQuang) [reply src/Infrastructure/Persistence/Configurations/EntryPermissionConfiguration.cs:11]: composite key

**Pull request merged**